### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this milestone, the following was achieved:

 Created forms to perform the following functions:
    - Creates a Post on behalf of the current_user.
    - Create a comment on behalf of the current_user.
    - Allow Users to add likes to Posts.

### CODE REVIEW

**Review by enaburekhan — COMMENTED** [review]
Hi Wuyepabdul, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with no descriptive readme. Kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to add a descriptive readme to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,4 +8,21 @@ def show
...
+    if post.save
+      redirect_to user_post_path(post.author.id, post.id), notice: 'Published successfully!'
+    else
+      flash.now[:alert] = 'Failed to publish post!'
+    end
+  end
+
+  def post_params
+    params.require(:post).permit(:title, :text)
+  end
```
Comment (AshakaE) [inline app/controllers/posts_controller.rb:29]: - [x] Kindly make `post_params` private.
see reference for [strong params](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        if comment.save
+
+          redirect_to user_post_path(post.author.id, post.id), notice: 'Comment added!'
+        else
+          flash.now[:error] = 'Faild to send comment!'
+          redirect_to user_post_path(post.author.id, post.id), alert: 'Failed to add comment!'
+        end
+      end
+    end
+  end
```
Comment (AshakaE) [inline app/controllers/comments_controller.rb]: - [x] Kindly initialize new comments with a method `new`

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @wuyepabdul  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @wuyepabdul   🙋‍♀️, 

# STATUS APPROVED ✔

Thanks for making the requested changes 👍 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**